hdlparse/vhdl_parser.py

```diff
@@ -750,11 +766,7 @@ def is_array(self, data_type):
         Returns:
           True if ``data_type`` is a known array type.
         """
-
-        # Split off any brackets
-        data_type = data_type.split('[')[0].strip()
-
-        return data_type.lower() in self.array_types
+        return data_type.name.lower() in self.array_types
```

Comment:
Passing in `vhdl_ex.is_array("unsigned")` now fails because string does not have an attribute name.